pkg/streamingpromql/planning.go

```diff
@@ -812,9 +811,9 @@ func NewMaximumSupportedVersionQueryPlanVersionProvider() QueryPlanVersionProvid
 }
 
 type staticQueryPlanVersionProvider struct {
-	version uint64
+	version planning.QueryPlanVersion
 }
 
-func (s *staticQueryPlanVersionProvider) GetMaximumSupportedQueryPlanVersion(_ context.Context) (uint64, error) {
+func (s *staticQueryPlanVersionProvider) GetMaximumSupportedQueryPlanVersion(ctx context.Context) (planning.QueryPlanVersion, error) {
```

Comment:
Same comment like in the previous PR. If we don't need context, should we remote this from parameter?

Reply:
Are you suggesting to remove the context parameter from the method definition in the interface, or to rename this to `_` here?